core/domain/stats_domain.py

```diff
@@ -41,6 +41,158 @@
 MIGRATED_STATE_ANSWER_TIME_SPENT_IN_SEC = 0.0
 
 
+class ExplorationStats(object):
+    """Domain object representing analytics data for an exploration."""
+
+    def __init__(self, exp_id, exp_version, num_actual_starts, num_completions,
```

Comment:
break on the start of paranthesis if you have to.

Reply:
Done.